server/utils/is-protected-entity.js

```diff
@@ -2,6 +2,7 @@
 
 const isProtectedEntity = (entity, rules) =>
   rules.some((rule) => {
+    if(rule.length !== 3) return false
```

Comment:
@BorysShulyak Is should not be necessary to check the `rule.length` here because the plugin will validate the plugin config for that kind of usage when Strapi starts up.

Reply:
@mattmilburn Removed this statement.